change/@fluentui-react-popover-fec81255-7a72-4ab1-9324-c96c3961e502.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33424&commitId=675b3f74a35c7e8d1d93144ecf1760fbee80670a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|7| Changed|
|Avatar Converged.size+image+badge.chromium.png|1| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|975| Changed|
|Drawer.overlay drawer full.chromium.png|1154| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2220| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>